sdk/CMakeLists.txt

```diff
@@ -108,6 +108,9 @@ add_custom_command(
     ${CMAKE_SOURCE_DIR}/third-party/flatcc/bin/flatcc -cwr -o
     ${_program_schema__include_dir}/executorch/sdk/etdump
     ${_etdump_schema__srcs}
+  COMMAND
```

Comment:
I had just been wrestling with the flatcc cmake system, so I see why this is necessary. It seems like a reasonable short-term workaround, but it's magical enough that it deserves a comment to explain why it's here, and a TODO to remove it.

```suggestion
  # TODO(dbort): flatcc installs its files directly in its source directory
  # instead of under CMAKE_BINARY_DIR, and it has no options to avoid
  # doing this. We build flatcc twice in the executorch build: once to get
  # the `flatcc` host commandline tool, and once to get the (potentially
  # cross-compiled) target runtime library. The host build will put its outputs
  # in the source tree, making the cross-compiling target build think that
  # the outputs have already been built. It will then try to link against the
  # host-architecture libraries, failing when cross-compiling. To work around
  # this, delete the host outputs after running this command (which only runs
  # when setting up the cmake files, not when actually building). This leaves
  # room for the target build to put its own files in the source tree. We should
  # try to remove this hack, ideally by submitting an upstream PR that adds an
  # option to change the installation location.
  COMMAND
```

Reply:
Thanks for writing this piece of comment, Dave. It's very detailed and helpful.

Reply:
Thanks for your suggestion. It's very helpful.